### PR TITLE
Fix gas price input

### PR DESCRIPTION
**Pull request by tomsmalley**

### CODE REVIEW

File: frontend/src/Frontend/UI/DeploymentSettings.hs [hunk 1]
```diff
@@ -479,23 +482,18 @@ uiMetaData m mTTL mGasLimit = do
...
-      parseAndScaleWith
-        :: (GasPrice -> GasPrice)
-        -> InputElement er d t
-        -> Event t Text
-      parseAndScaleWith f =
-        fmap (showGasPrice . f) . eParsedGasPrice . _inputElement_input
-
-      eParsedGasPrice :: Event t Text -> Event t GasPrice
-      eParsedGasPrice = fmapMaybe (readPact (GasPrice . ParsedDecimal))
+      -- Parse the gas price, returning the price and if it was rounded
```
Comment (alexfmpe) [inline frontend/src/Frontend/UI/DeploymentSettings.hs]: `and if it was rounded` ?

Comment (tomsmalley) [reply frontend/src/Frontend/UI/DeploymentSettings.hs]: `(price, was rounded)` I should probably reword that

Comment (alexfmpe) [reply frontend/src/Frontend/UI/DeploymentSettings.hs]: That's `parseDecimal :: Text -> Maybe (Decimal, Bool)` this one is `parseGasPrice :: Text -> Maybe GasPrice
`

Comment (tomsmalley) [reply frontend/src/Frontend/UI/DeploymentSettings.hs]: Oops :smile:

Comment (tomsmalley) [reply frontend/src/Frontend/UI/DeploymentSettings.hs]: I removed it, it seems pointless now.

**Review by mankyKitty — APPROVED** [review]

**Pull request merged**